styles/config/vocabularies/Canonical/accept.txt

```diff
@@ -44,11 +56,15 @@ VMware
 ZooKeeper
 config
 downtime
+eksctl
 keystore
+(?i)namespace
+parallelization
```

Comment:
```suggestion
parallelisation
```

There's an interesting conversation to be had about specific terms accepted in IT lexicon as US spelling - but this is the UK version.